analyses/snv-callers/run_caller_consensus_analysis-pbta.sh

```diff
@@ -8,60 +8,72 @@
 set -e
 set -o pipefail
 
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
 # The sqlite database made from the callers will be called:
-dbfile=scratch/snv_db.sqlite
+dbfile=../../scratch/snv_db.sqlite
 
 # Designate output file
-consensus_file=analyses/snv-callers/results/consensus/pbta-snv-consensus-mutation.maf.tsv
+consensus_file=results/consensus/pbta-snv-consensus-mutation.maf.tsv
 
 # BED and GTF file paths
-cds_file=scratch/gencode.v27.primary_assembly.annotation.bed
-wgs_bed=scratch/intersect_strelka_mutect_WGS.bed
+cds_file=../../scratch/gencode.v27.primary_assembly.annotation.bed
+wgs_bed=../../scratch/intersect_strelka_mutect_WGS.bed
 
 # Set a default for the VAF filter if none is specified
 vaf_cutoff=${OPENPBTA_VAF_CUTOFF:-0}
 
+# If running during data generation, we want to use the BASE histologies file
+run_for_release=${OPENPBTA_BASE_RELEASE:-0}
```

Comment:
For consistency with other modules, consider all caps `RUN_FOR_RELEASE`

Reply:
This is for consistency _within this module_. Otherwise, I'd agree.